core/src/main/scala/org/bykn/bosatsu/Normalization.scala

```diff
@@ -63,7 +84,29 @@ object NormalExpression {
   }
 }
 
-sealed abstract class NormalPattern {}
+sealed abstract class NormalPattern {
+  def escapeString(unescaped: String) = unescaped.replaceAll("""([\\'])""", """\\$1""")
```

Comment:
there is a well tested string escaping code here:

https://github.com/johnynek/bosatsu/blob/master/core/src/main/scala/org/bykn/bosatsu/StringUtil.scala#L41

Is this doing something else special? Actually, can we comment what it is doing?

Reply:
I'm definitely happier to use something well tested than a regex hack. This is just attempting to escape `'` to `\'` and also escape `\` to `\\`